frontend/src/APIClients/CamperAPIClient.ts

```diff
@@ -109,11 +111,23 @@ const getCampersByChargeIdAndSessionId = async (
   }
 };
 
+const registerCampers = async (
+  campers: CreateCamperRequest[],
+): Promise<CheckoutUrl> => {
+  const { data } = await baseAPIClient.post(`/campers/register`, campers, {
```

Comment:
There are some changes to this backend endpoint: 
1. We now pass in an array of sessionIds to support multi-session registration
2. The actual campers array that Thomas created has changed so that the camper data that we send to the backend doesn't include the charges, id, and campSession fields. Just wanted to verify if those changes work with this.

Do you need the charges field on the campers for the stripe processing? I thought it would be cleaner for the backend to add the charges field than the frontend trying to calculate it